explainer.md

```diff
@@ -499,6 +512,8 @@ interface VRDevice : EventTarget {
 
   Promise<boolean> supportsSession(VRSessionCreateParametersInit parameters);
   Promise<VRSession> requestSession(VRSessionCreateParametersInit parameters);
+
+  Promise<void> ensureContextCompatibility(WebGLRenderingContext context);
```

Comment:
Good catch. I swear I had already done that.

I don't _think_ there's a problem with using `WebGLRenderingContextBase` here, even if it isn't Javascript exposed. The IDLs should still parse correctly. (But if not, then yes. A union would work just fine.)